test/lint/lint-all.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2017-2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# This script runs all test/lint/lint-* files, and fails if any exit
+# with a non-zero status code.
+
+from glob import glob
+from pathlib import Path
+from subprocess import run
+
+exit_code = 0
+mod_path = Path(__file__).parent
+for lint in glob(f"{mod_path}/lint-*"):
```

Comment:
```suggestion
for lint in glob(f"{mod_path}/lint-*.py"):
```

wondering if we can make this `py`

Reply:
If yes, we can run the subtests with `sys.executable` instead of calling out to `python3`

Reply:
It can be `*.py` after #24840. I'd also like a lint to prevent someone adding a non-Python linter again :smile: Though it's possibly out of scope of this PR.